apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -11,9 +13,16 @@ var fromApp = require(appRoot + '/shared/test/integration/require_from_app');
 function ConversationView(client) {
   this.client = client;
   this.accessors = new ConversationAccessor(client);
+  this.composerAccessors = new ComposerAccessor(client);
+  this.menuAccessors = new MenuAccessor(client);
 }
 
 ConversationView.prototype = {
+  // TODO Move these constants to marionette
```

Comment:
Okay, I went ahead and filed one https://bugzilla.mozilla.org/show_bug.cgi?id=1207516 :)